skfda/preprocessing/dim_reduction/feature_extraction/__init__.py

```diff
@@ -1,5 +1,9 @@
-"""Feature extraction."""
-from ._ddg_transformer import DDGTransformer
-from ._fda_feature_union import FDAFeatureUnion
-from ._fpca import FPCA
-from ._per_class_transformer import PerClassTransformer
+import warnings
+
+from ... import feature_construction
```

Comment:
At least FPCA should be imported here to preserve old behaviour, not `feature_construction`.